wifiphisher/data/phishing-pages/device-modal/config.ini

```diff
@@ -0,0 +1,6 @@
+[info]
+Name: Device Modal
+Description: Shows a login modal based on the victim's device/browser.
```

Comment:
Name: Device Modal
Description: Shows the iOS/Android WiFi login modal on phones and tablets. Shows a Chrome login modal for a .htaccess restricted page on desktops/laptops.